test/e2e-test/pages/ask-for-more-time/ask-for-more-time.ts

```diff
@@ -0,0 +1,21 @@
+import { paths } from '../../../../app/paths';
+
+const config = require('config');
+
+const testUrl = config.get('testUrl');
+
+module.exports = {
+  askForMoreTime(I) {
+    Then(/^I click Ask for more time$/, async () => {
+      await I.amOnPage(testUrl + paths.askForMoreTime);
+    });
+
+    Then(/^I should see the ask-for-more-time page$/, async () => {
+      await I.seeInCurrentUrl(paths.askForMoreTime);
+    });
+
+    When(/^I click save and continue$/, async () => {
+      await I.click('Save and continue');
```

Comment:
Remove this, we already have functions for clicking buttons and links.

Use instead `I click "Save and continue" button`